mmr-lib/Templates/T4Extensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace MMRando.Templates
+{
+    internal static class T4Extensions
```

Comment:
Wouldn't `TypeExtensions` be a more appropriate name?

Reply:
No. This will be removed anyway since it isn't needed